crates/pathfinder/src/sequencer.rs

```diff
@@ -622,6 +670,9 @@ pub mod test_utils {
         pub static ref VALID_KEY: StorageAddress = StorageAddress::from_hex_str("0x0206F38F7E4F15E87567361213C28F235CCCDAA1D7FD34C9DB1DFE9489C6A091").unwrap();
         pub static ref VALID_KEY_DEC: String = crate::rpc::serde::starkhash_to_dec_str(&VALID_KEY.0);
         pub static ref VALID_CALL_DATA: Vec<CallParam> = vec![CallParam::from_hex_str("0x4d2").unwrap()];
+        // TODO these values are for external.integration.starknet.io and should be fixed when goerli gets to 0.9.0
```

Comment:
I don't have a valid value for Goerli yet.

Reply:
You could choose any contract hash on Goerli I guess. Just pick one from our DB.

Reply:
Oh my, for the 100th time I forgot that class hash is just contract hash.